### PR TITLE
Moved fullCountries to background

### PR DESCRIPTION
**Pull request by yuubd**
Moved fullCountries.json to background
Added temServer to fetch coordinates
Refactored related logic

### CODE REVIEW

File: src/tmpServer.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+'use strict';
+import { features as countries } from './fullCountries.json';
```
Comment (jieun-lee) [inline src/tmpServer.js:2]: this isn't necessarily related to this specific line but... I think we agreed to remove some countries with insufficient data (UMI - no latlon data / HMD - no capital, continent, population 0), should that change go in with this fix?

Comment (yuubd) [reply src/tmpServer.js:2]: We will remove the in the blitz.

File: src/frontEnd/src/mainLayout/components/nameSectionComponent/searchBarComponent/searchBarComponent.tsx [hunk 2]
```diff
@@ -25,11 +25,12 @@ function SearchBarComponent(props: { initStr: string, size: number, setSearching
...
-        for (let index in countries) {
-            if (regex.test(countries[index].properties.ADMIN.toLowerCase())) {
+        for (let idx = 0; idx < NUMBER_OF_COUNTRIES; idx++) {
+            const countryName = getCountryName(idx).toLowerCase();
+            if (regex.test(countryName)) {
                 matched.push({
-                    index: +index,
-                    name: countries[index].properties.ADMIN
+                    index: +idx,
+                    name: countryName
```
[the remark this answers is not in the document]
Comment (yuubd) [reply src/frontEnd/src/mainLayout/components/nameSectionComponent/searchBarComponent/searchBarComponent.tsx:33]: Capitalizing as we show makes sense to me.

[the remark this answers is not in the document]
Comment (yuubd) [reply src/frontEnd/src/mainLayout/components/nameSectionComponent/searchBarComponent/searchBarComponent.tsx:33]: Fixed it

**Pull request merged**